### DISCUSSION TITLE
Help, I want to implement hundreds of loops of jobs. Request Optimal configuration

### DISCUSSION THREAD

**Discussion by windsOne** [quartznet/quartznet#772] [General]
Quartz: 2.6.2.0
I want to implement hundreds of loops of jobs. Request Optimal configuration

**Comment by lahma** [comment]
Quartz should be quite optimal. Should work fine unless you find a bottleneck. I don't think there's much to work with in this issue.

**Comment by windsOne** [comment]
![image](https://user-images.githubusercontent.com/14890297/60568188-b5ae3f80-9d9e-11e9-9e88-9cbf80c69e8a.png)
I have to set the expiration time to 20 seconds to avoid getting misfire  a lot of data.

**Comment by windsOne** [comment]
@lahma

**Comment by lahma** [comment]
What are your jobs doing, if the run long it's natural that 80 threads won't do it. Also your logging might be the bottleneck (LoggingTriggerPlugin), try running without it.

**Comment by windsOne** [comment]
@lahma  how can  close logging
